keyboards/handwired/ferris/config.h

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2020 Pierre Chevalier <pierrechevalier83@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xC2AB
+#define PRODUCT_ID 0x0000
+#define DEVICE_VER 0x0001
+#define MANUFACTURER Pierre
+#define PRODUCT Ferris the keeb
+#define DESCRIPTION A minimalistic 34 - keys split keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 10
+
+#define MATRIX_ROWS_PER_SIDE (MATRIX_ROWS / 2)
+#define MATRIX_COLS_PER_SIDE (MATRIX_COLS / 2)
+
+#define UNUSED_MCU 14
+#define UNUSED_MCP 7
+
+// wiring
+#define MATRIX_ROW_PINS_MCU \
+    { B3, B2, B1, F0 }
+#define MATRIX_COL_PINS_MCU \
+    { D6, D7, B4, B5, B6 }
+#define UNUSED_PINS_MCU \
+    { B0, B7, C6, C7, D2, D3, D4, D5, E6, F1, F4, F5, F6, F7 }
+#define MATRIX_ROW_PINS_MCP \
+    { B0, B1, B2, B3 }
+#define MATRIX_COL_PINS_MCP \
+    { A0, A1, A2, A3, A4 }
+#define UNUSED_PINS_MCP \
+    { B4, B5, B6, B7, A5, A6, A7 }
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* define if matrix has ghost (lacks anti-ghosting diodes) */
+//#define MATRIX_HAS_GHOST
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+// Set the mouse settings to a comfortable speed/accuracy trade-off,
+// assuming a screen refresh rate of 60 Htz or higher
+// The default is 50. This makes the mouse ~3 times faster and more accurate
+#define MOUSEKEY_INTERVAL 16
+// The default is 20. Since we made the mouse about 3 times faster with the previous setting,
+// give it more time to accelerate to max speed to retain precise control over short distances.
+#define MOUSEKEY_TIME_TO_MAX 40
+// The default is 300. Let's try and make this as low as possible while keeping the cursor responsive
+#define MOUSEKEY_DELAY 100
+// It makes sense to use the same delay for the mouseweel
+#define MOUSEKEY_WHEEL_DELAY 100
+// The default is 100
+#define MOUSEKEY_WHEEL_INTERVAL 50
+// The default is 40
+#define MOUSEKEY_WHEEL_TIME_TO_MAX 100
+
+// Pick good defaults for enabling homerow modifiers
+#define TAPPING_TERM 200
+#define PERMISSIVE_HOLD
+#define IGNORE_MOD_TAP_INTERRUPT
+#define TAPPING_FORCE_HOLD
```

Comment:
Given that all these are generally a user-preference thing, I'd probably advocate for moving these into a `config.h` inside the default keymap, so that people can use their own preferred values without having them overridden here.

Reply:
I see where you're coming from, although I thought that this was a bit of a special situation as the keyboard has only 34 keys, which makes use of homerow modifiers almost a necessary thing. From seeing the discussion in the splitkb discord, almost everyone ends up with these settings or something quite close after trying the homerow modifiers with default settings and having issues. While the default keymap is definitely a place where these could leave, I think people can also override them in their own keymaps, but with these here, they will have to change less things to get to a workable default. In my opinion, almost no-one will use the default keymap in the long run even if I make it as good as can be (not the case yet, but I have plans for subsequent PRs) as too much of how one interacts with a computer is a matter of personal preference. So for settings like this where I strongly believe they will give a better experience by default to most people, I find this location a bit more obvious. Of course, this entire paragraph is quite opinionated, so if you still think they should go to the default keymap after hearing how I see things, I'll move them over there :smile: